app/src/main/java/edu/uc/group/rankine/dto/ObjectSet.kt

```diff
@@ -24,19 +28,14 @@ data class ObjectSet(var elements: ArrayList<ElementObject> = ArrayList(), var f
     }
 
     /**
-     * Modify an element <inputObject> in the ObjectSet with the name and attributes of element <newObject>
+     * Modify an element <inputObject> in the ObjectSet with the name and attributes of element <newObject> at its original index
```

Comment:
Is this needed functionality?

Since objects are stored by reference in a collection, they will update automatically.